code/routing/FluentRootURLController.php

```diff
@@ -99,4 +99,22 @@ public function handleRequest(SS_HTTPRequest $request, DataModel $model = null)
 		$this->popCurrent();
 		return $result;
 	}
+    
+    /**
+     * 
+     * With Translatable installed, don't pre-append the locale to the homepage 
+     * URL.
+     * 
+     * @return string
+     * @see {@link Translatable::get_homepage_link()}.
+     */
+    public static function fluent_homepage_link($localeURL) {
+        $homepageLink = parent::get_homepage_link();
+        if(class_exists('Translatable')) {
```

Comment:
An in-code comment would help with future debugging as well.

```
// Here we don't append the prefix because <reason>. When translatable is installed <some change in behaviour> so this code is awesome and trustworthy because <comforting reassurance>
```